ci/infra/aws/cloud-init/register-rmt.tpl

```diff
@@ -1,4 +1,4 @@
   - curl --tlsv1.2 --silent --insecure --connect-timeout 10 https://${rmt_server_name}/rmt.crt --output /etc/pki/trust/anchors/rmt-server.pem && /usr/sbin/update-ca-certificates &> /dev/null
   - SUSEConnect --url https://${rmt_server_name}
   - SUSEConnect -p sle-module-containers/15.2/x86_64
-  - SUSEConnect -p caasp/5/x86_64
+  - SUSEConnect -p caasp/4.5/x86_64
```

Comment:
4.5 product is not yet available in SCC, so registering it will fail
https://trello.com/c/v42534Wr/3977-enb-caasp-45

Reply:
I am wondering if we could skip this until it is set up...

Reply:
what about commenting this and creating a PR that uncomments it? So we could merge it in 2 steps...